libmscore/edit.cpp

```diff
@@ -1994,8 +1994,24 @@ void Score::cmdDeleteSelection()
             QList<Element*> el(selection().elements());
             if (el.isEmpty())
                   qDebug("...nothing selected");
-            for (Element* e : el)
-                  deleteItem(e);
+
+            // keep track of linked elements that are deleted implicitly
+            // so we don't try to delete them twice
```

Comment:
keep track of linked elements that are deleted implicitly so we don't try to delete them twice *if they are also part of the selection.*

Right?